### ISSUE TITLE
Crash after create a new channel 1 to 1

### ISSUE THREAD

**Issue by maxmaxandr** [GetStream/stream-chat-android#481]
I've got crash when trying to create a new channel 1 to 1

```
E/AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-2
     java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter role
        at io.getstream.chat.android.client.models.Member.<init>(Unknown Source:8)
        at io.getstream.chat.android.client.models.Member.<init>(Member.kt:19)
        at io.getstream.chat.android.livedata.entity.MemberEntity.toMember(MemberEntity.kt:41)
        at io.getstream.chat.android.livedata.entity.ChannelEntity.toChannel(ChannelEntity.kt:98)
        at io.getstream.chat.android.livedata.ChatDomainImpl.selectAndEnrichChannels$library_release(ChatDomainImpl.kt:657)
        at io.getstream.chat.android.livedata.ChatDomainImpl$selectAndEnrichChannels$2.invokeSuspend(Unknown Source:12)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**Comment by tschellenbach** [comment]
You're creating the channel server side right? @tbarbugli any idea why role would be null?

**Comment by tschellenbach** [comment]
@maxmaxandr you reported quite a few issues. we will, of course, fix them all, but I wanted to check in on priorities, which ones are holding you back most?

**Comment by maxmaxandr** [comment]
> You're creating the channel server side right? @tbarbugli any idea why role would be null?

yes. Server side

**Comment by maxmaxandr** [comment]
> @maxmaxandr you reported quite a few issues. we will, of course, fix them all, but I wanted to check in on priorities, which ones are holding you back most?

All bugs that cause crashes. We need to show application to client and crashes are worst for us.

**Comment by psylinse** [comment]
Ran into same issue: https://github.com/GetStream/stream-chat-android/issues/486

I'm creating everything client side, including channels, so not sure @maxmaxandr's has anything to do with it. 

Here's a broken repo (that was working on `4.0.13`) that might help track this down: https://github.com/psylinse/the-stream-android/

**Comment by psylinse** [comment]
This is happening inconsistently on `4.2.7`. Seems a little better but 50% of the time it crashes with this.

**Comment by maxmaxandr** [comment]
Hi, any plan when it will be fixed?

**Comment by elevenetc** [comment]
@maxmaxandr fixed in the latest release: `4.2.9-beta-1`. From now on `member.role` is optional.
